data/rulesets.js

```diff
@@ -63,6 +63,23 @@ exports.BattleFormats = {
 	},
 	pokemon: {
 		effectType: 'Banlist',
+		validateTeam: function (team, format) {
+			var problems = [];
+			// ----------- legality line ------------------------------------------
```

Comment:
I'd move `var problems` after the legality line, and just return `null` in the legality check.